lib/README.md

```diff
@@ -0,0 +1,8 @@
+# Configuration Structure C Library
+
```

Comment:
It would be great if you  could please add the svg diagram you showed us today, many thanks in advance